src/simulator/render.py

```diff
@@ -125,4 +125,10 @@ def render_toolbar(screen, time_elapsed, buttons):
     toolbar_rect = pygame.Rect(0, screen.get_height()-TOOLBAR_HEIGHT,screen.get_width(),TOOLBAR_HEIGHT)
     pygame.draw.rect(screen, pygame.Color(80,80,80), toolbar_rect)
     render_time(screen,toolbar_rect, time_elapsed)
-    render_buttons(screen, buttons)
\ No newline at end of file
+    render_buttons(screen, buttons)
+
+def render_title(screen): 
+    # draw title and version
+    FONT = pygame.font.SysFont("Segoe UI", 15, bold=True, italic=False)
+    text_surface = FONT.render(f"Concurent Traffic. Version s.1.0.", True, (255, 255, 255))
```

Comment:
wow! excellent work
let's have the text be formatted: "Concurrent Traffic vX.X.X"
also could we have three constants MAJOR, MINOR, PATCH that are defined in main.py file, and we will have to import it in.
refer to https://realpython.com/python-constants/